README.md

```diff
@@ -134,9 +134,14 @@ The following inputs should be provided for every organization workflow.
         documentation: "README.md"
 ```
 
+## Original event information
+
+The complete event generated in repository that triggered the workflow is available at `${{ github.event.client_payload.data.<event_field_path> }}`.
+Checkout [Github's docs](https://docs.github.com/en/developers/webhooks-and-events) to see the information available in the event.
```

Comment:
This app is triggered by the push event, so it makes sense to link to the `push` anchor on the event payload page: 

https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#push

Reply:
I do have uses cases for other events, but I'll send that proposal in a new PR.